printf/src/arg.rs

```diff
@@ -169,6 +171,20 @@ impl<'a, T> ToArg<'a> for &'a *const T {
     }
 }
 
+#[cfg(feature = "bstr")]
+impl<'a> ToArg<'a> for &'a BStr {
+    fn to_arg(self) -> Arg<'a> {
```

Comment:
> Can formatx! and wgettext! co-exist?

probably but it seems more intuitive to first change the format string syntax without changing string encoding.